### DISCUSSION TITLE
Running nativewind on web and ios

### DISCUSSION THREAD

**Discussion by wlumumba** [nativewind/nativewind#469] [Q&A]
So I got nativewind on web to work and show styling when I run `npx expo web`. I had to create a styles.css, fill it with tailwind directives and import it in my App.js - after setting up Tailwind CSS like normal.

Problem is when I import the styles.css, my iOS simulator throws a syntax error and says cannot resolve the styles module. Now, if I remove the import then none of my classnames get applied in web. I'm very new to development, so any guidance would be appreciated!

Error below
```
None of these files exist:
  * ../styles.css(.native|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.json|.native.json|.json)
  * ../styles.css/index(.native|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.json|.native.json|.js
```

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
